### PR TITLE
Revenue be

### PR DESCRIPTION
**Pull request by cuongbui1203**
## Related Tickets:
- [#73366](https://edu-redmine.sun-asterisk.vn/issues/73366)

## WHAT this PR does?
Tạo logic lấy dữ liệu doanh thu

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by cuongbui1203** [comment]
ready

File: database/seeders/UserSeeder.php [hunk 1]
```diff
@@ -68,4 +68,4 @@ public function run()
             'role_id' => 3,
         ]);
     }
-}
+}
```
Comment (manhtd-1083) [inline database/seeders/UserSeeder.php:71]: eof

File: resources/js/Pages/Admin/ListOrder.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (manhtd-1083) [inline resources/js/Pages/Admin/ListOrder.js:1]: ? empty line

**Comment by cuongbui1203** [comment]
ready

**Pull request merged**